### PR TITLE
Fixing problem with @ char in paths during pack (#1348)

### PR DESCRIPTION
**Pull request by Vidarls**
ref: #1348

### CODE REVIEW

**Comment by forki** [comment]
the travis fail is unrelated.
thx

File: integrationtests/Paket.IntegrationTests/PackSpecs.fs [hunk 1]
```diff
@@ -17,4 +18,22 @@ let ``#1234 empty assembly name``() =
     with
     | exn when exn.Message.Contains("PaketBug.dll") -> ()
 
-    File.Delete(Path.Combine(scenarioTempPath "i001234-missing-assemblyname","PaketBug","paket.template"))
\ No newline at end of file
+    File.Delete(Path.Combine(scenarioTempPath "i001234-missing-assemblyname","PaketBug","paket.template"))
+
+[<Test>]
+let ``#1235 npm type folder names`` () =
```
Comment (forki) [inline integrationtests/Paket.IntegrationTests/PackSpecs.fs]: Awesome.

**Comment by Vidarls** [comment]
ok, good. Was scratching my head severely now

**Comment by Vidarls** [comment]
Quick question, what is the logic behind the # 1234 type naming in the integration tests? I just did the above test number +1, but now I am starting to think it might be references to github issues?

**Comment by forki** [comment]
yep test name should be "#1348 ..."

**Comment by forki** [comment]
this allows us to find the issue from the test (when we break it in the future)

**Comment by Vidarls** [comment]
fixing...

**Comment by Vidarls** [comment]
fixed

**Pull request merged**